atomsci/ddm/utils/struct_utils.py

```diff
@@ -510,7 +508,7 @@ def canonical_tautomers_from_smiles(smiles):
         (list of str) : List of SMILES strings for the canonical tautomers.
     """
     taut_enum = rdMolStandardize.TautomerEnumerator()
-    if type(smiles) == str:
+    if isinstance(smiles, str):
```

Comment:
test?

Reply:
These tests should cover it. https://github.com/ATOMScience-org/AMPL/commit/8246be5a91e0e4999343198956c5085573903396